src/main/java/core/basesyntax/RightTriangle.java

```diff
@@ -0,0 +1,24 @@
+package core.basesyntax;
+
+public class RightTriangle extends Figure {
```

Comment:
Don't begin class or method implementation with an empty line. Please remove the empty line at the beginning of the class definition.